README.md

```diff
@@ -47,8 +47,35 @@ For a more detailed example, see `usage.py`.
 
 ### Quickstart (R)
 
-This section will be added soon.
+First, install the package from GitHub (the package is not yet available via CRAN):
+```
+remotes::install_github("plotly/dash-deck")
+```
+
+then, create your component and add it to your layout:
+
+```r
+library(dash)
+library(dashVtk)
+library(dashHtmlComponents)
+
 
+layout <- htmlDiv(
+    style = list("width" = "100%", "height" = "calc(100vh - 16px)"),
+    children = View(list(
```

Comment:
In R, `View` is a function exported from the namespace of the `utils` package, so this won't work :upside_down_face: I think you might have meant instead

```suggestion
    children = dashVtk::View(list(
```

However, I would _strongly_ discourage overloading `View`, as this function is essentially loaded in every new R session. 

My suspicion is that CRAN will likely reject the package with the function named as it is, because of namespace collisions. (I would, if I were them! This is not obvious unless you're an R user, so no worries for not catching this.)

Is it possible to rename `View`?

Reply:
Yes to rename `View` but to what?

`VtkView`?

Reply:
If the capitalization isn't essential for the package, making `view` (and other functions) lower case would sidestep this issue since object names are case-sensitive in R.

Reply:
Hum, I thought we had to be Capitalize (by convention) on the react side. But maybe we can lowercase them? @xhlulu  or else what do you think?

Reply:
@jourdain Would it be easier to make all the function names lower case instead? It's a bit more idiomatic to use lower case names for functions in R anyhow, it's not required but an informal convention reflected in current style recommendations:

http://adv-r.had.co.nz/Style.html
https://journal.r-project.org/archive/2012-2/RJournal_2012-2_Baaaath.pdf

Exceptions include classes (like the `Dash` R6 class :slightly_smiling_face:) and a few core functions like `Map`, `Reduce`, `View`, etc.

Reply:
I don't see much issue with that except that in other language (py) it may seems strange. But nothing wrong about that, just not the habit compare to the other dash library.

Reply:
It's OK to have components with lower case or camel cased names, e.g. `dccGraph`, `htmlDiv` are existing functions that reference component classes in Dash for R. 

As for the `children`, I'd guess that as long as the `view` function returns a valid object, the content will be serialized into JSON exactly as before when it was named `View`.

Reply:
> I don't see much issue with that except that in other language (py) it may seems strange. But nothing wrong about that, just not the habit compare to the other dash library.

@jourdain I think this is exactly correct -- the naming is slightly different, it might look odd to a Python user, but conversely would seem ordinary to an R user.

Reply:
Is there a way (config/setting) to have those component lowercase in R and Capitalized in Python?

Reply:
@jourdain we could also invoke the prefix option for `dash-generate-components`, so this would be instead

`vtkView`

which probably resembles most other Dash component libraries (besides, say, `dashCanvas`, which uses no prefix).

Reply:
Looks like you already have `vtk` as a prefix for both R and Julia, so I'm surprised that this would work as shown here... Also I notice that both in R (the `R` and `man` directories) and in Julia (`src/*.jl`) there are multiple versions of the same component build artifacts committed. We should really change the boilerplate so that it wipes these before building new ones in case either the prefix changes, a component name changes, or a component is deleted... but since we don't do that, can you clear these out manually please, and regenerate?

Reply:
I haven't tested the code but I can try to install R and run it.

Reply:
@alexcjohnson @rpkyle I will make a PR to wipe & regenerate the package (seems like the prefix is already fine by default, so I just need to correct the snippet). @jourdain is still working on the source code so I'll wait a bit in order to not disrupt his workflow